host/src/attribute.rs

```diff
@@ -204,8 +258,8 @@ impl<'d> AttributeData<'d> {
                     return Err(AttErrorCode::UnlikelyError);
                 }
 
-                *notifications = data[0] & 0x01 != 0;
-                *indications = data[0] & 0x02 != 0;
+                *notifications = data[0] & 0x00000001 != 0;
+                *indications = data[0] & 0x00000010 != 0;
```

Comment:
This value is not quite the same as 0x02, was this intentional?

Reply:
Oh yeah, I meant to do `0b`, not `0x`, my bad :sweat_smile: